samples/client/petstore/c/build-and-test.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+#install latest curl
+
+wget https://curl.haxx.se/download/curl-7.61.1.zip
+
+unzip curl-7.61.1.zip
+
+cd curl-7.61.1
+
+./configure
+
+make
+
+sudo make install
+
+cd ..
+
+cmake .
+
+make 
+
+./unit-PetAPI
```

Comment:
This only executes one unit-test, change this to ctest and it should execute all unit-tests.

Reply:
@PowerOfCreation thanks. I'll give it a try tomorrow.